ubuntu-headless-qt/README.md

```diff
@@ -0,0 +1,681 @@
+# RZG2L SBC board #
+This is the quick startup guide for RZG2L SBC board (hereinafter referred to as `RZG2L-SBC`).
```

Comment:
"This is the quick startup guide for RZG2L SBC board (hereinafter referred to as `RZG2L-SBC`)."

Change to something like

"This is the quick startup guide for RZG2L SBC board (hereinafter referred to as `RZG2L-SBC`) to develop on Ubuntu headless (no Desktop environment support)"